code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -718,6 +718,8 @@ datum/status_effect/stabilized/blue/on_remove()
 /datum/status_effect/stabilized/cerulean/tick()
 	if(owner.stat == DEAD)
 		if(clone && clone.stat != DEAD)
+			owner.visible_message(span_warning("[owner] begins to glow with brilliant light."))
+			sleep(30 SECONDS)
```

Comment:
what the fuck is htis

Reply:
the worst made PR of my rampage, by far
all it does is add a 30 second delay before it continues

Reply:
you need to use addtimer not sleep

Reply:
> you need to use addtimer not sleep

sleep is a global proc that adds delay, addtimer could also work, but then i'd need to make another proc for the timer to callback